upgrade/db/mysql/ezplatform-2.5.latest-to-3.0.0.sql

```diff
@@ -39,3 +39,21 @@ ALTER TABLE `ezeditorialworkflow_markings`
     ADD COLUMN `reviewer_id` INT(11),
     ADD COLUMN `result` TEXT;
 --
+
+-- EZEE-2988: Added availability for schedule hide --
+BEGIN;
+ALTER TABLE  `ezdatebasedpublisher_scheduled_version`
+CHANGE COLUMN `publication_date` `action_timestamp` INT(11) NOT NULL;
+ALTER TABLE  `ezdatebasedpublisher_scheduled_version`
+ADD COLUMN `action` VARCHAR(32);
+UPDATE `ezdatebasedpublisher_scheduled_version` SET `action` = 'publish';
+ALTER TABLE  `ezdatebasedpublisher_scheduled_version` CHANGE COLUMN `action` `action` VARCHAR(32) NOT NULL;
+COMMIT;
+--
+
+-- EZEE-2988: Added availability for schedule hide --
```

Comment:
This is the same ticket, so I don't see a point in separating it. If you feel the separate would be better than maybe just a simple `--` w/o redundant ticket mention?

BTW. any reason for committing before that last change below?

Reply:
After talk with @adamwojs we decided that for now we wan't to merge this changes in case od not much time.